src/ui/toolbox/menu.js

```diff
@@ -117,6 +117,30 @@ var menuComponentApi = {
         }
     },
 
+    /**
+     * It needs to find nearest visible item.
+     *
+     * @param {-1|1} inc - incrementor. -1 - navigate to the top,  1 - to the bottom.
+     *
+     * @returns {JQuery} element.
+     */
+    _findClosestVisibleItem(inc = 1) {
```

Comment:
Private functions don't need to start by an underscore, see the [frontend coding guide](https://hub.taotesting.com/articles/frontend/architecture/tao-frontend-coding-guide#general-rules).

Reply:
No problem, I'll remove it, but... Why? it's interesting to know the reason to mark a standard language notation as a bad practice.

Reply:
This is not a standard so far. The current state of the prefix notation for private members is still in stage 3, so not yet validated: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Class_fields#Private_fields

And it will not be the underscore.

You can also see some explanations here: https://stackoverflow.com/a/56205859

Moreover, we tend to not adopt too early new language features. For instance, we are still not allowing `async`/`await` yet. The reason why is we prefer to make sure all is properly supported and well mastered to avoid both too verbose transpilations and construction issues.

Reply:
I didn't mean language feature proposal, because to be truthy I hadn't heard about it. It's old javascript convention. An underscore before a name is used to specify that the name is to be treated as “private” by a programmer. 

From the second link:
> _ prefixed variable names are considered private by convention but are still public.